crates/rolldown/src/ast_scanner/const_eval.rs

```diff
@@ -32,6 +35,10 @@ impl<'ast> GlobalContext<'ast> for ConstEvalCtx<'_, 'ast> {
     &self,
     reference_id: oxc::semantic::ReferenceId,
   ) -> Option<oxc_ecmascript::constant_evaluation::ConstantValue<'ast>> {
+    // If there is an override function, return its result whatever.
+    if let Some(res) = self.overrode_get_constant_value_from_reference_id.map(|f| f(reference_id)) {
+      return res;
+    }
```

Comment:
Is this meant to be like below? or is the comment wrong?
```suggestion
    // If there is an override function, return its result whatever.
    if let Some(f) = self.overrode_get_constant_value_from_reference_id {
      return f(reference_id);
    }
```

Reply:
Maybe using a separate struct makes the code simple?

Reply:
You suggesion seems simpler.